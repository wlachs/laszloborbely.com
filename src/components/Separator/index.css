@import '../global.css';

.separator {
  border-bottom: 1px solid var(--primary-color);
}