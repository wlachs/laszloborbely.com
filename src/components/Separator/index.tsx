import './index.css';

function Separator() {
	return <div className='separator w-100 mb-3' />;
}

export default Separator;
