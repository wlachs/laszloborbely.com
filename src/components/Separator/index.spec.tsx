import {cleanup, render} from '@testing-library/react';
import {afterEach, describe, expect, it} from 'vitest';
import Separator from './index.tsx';

describe('Separator component', () => {
	afterEach(() => {
		cleanup();
	});

	it('should match snapshot', () => {
		const {container} = render(<Separator/>);
		expect(container).toMatchSnapshot();
	});
});
