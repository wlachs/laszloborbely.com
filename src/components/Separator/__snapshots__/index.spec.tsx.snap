// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Separator component > should match snapshot 1`] = `
<div>
  <div
    class="separator w-100 mb-3"
  />
</div>
`;
